src/api/telemetry/TelemetryAPI.js

```diff
@@ -298,6 +312,61 @@ export default class TelemetryAPI {
         }
     }
 
+    /**
```

Comment:
Thanks for providing some docs for the new API here. Could you please also update [the Telemetry Provider section of our API doc](https://github.com/nasa/openmct/blob/master/API.md#telemetry-providers)?

Reply:
tracking this in: #6152